src/components/pages/VaultDetail/index.tsx

```diff
@@ -62,6 +63,13 @@ export const VaultDetail = (): JSX.Element => {
 
   return (
     <div className="p-4 space-y-8">
+      <Helmet>
+        <title>DAO Books - vault details</title>
+        <meta
```

Comment:
meta description could be added to the App.tsx and remove it from all pages as they are essentially the same.